packages/ckeditor5-table/src/tablecolumnresize/utils.js

```diff
@@ -365,3 +365,21 @@ export function removeColumnResizerElements( viewWriter, viewCell ) {
 
 	viewWriter.remove( viewTableColumnResizerElement );
 }
+
+// Enables/disables resizers in specified view.
+//
+// @param {module:ui/view~View} view
+// @param {Boolean} isEnabled
+export function setResizersEnablement( view, isEnabled ) {
+	view.change( viewWriter => {
+		for ( const root of view.domRoots.values() ) {
+			const viewRoot = view.domConverter.mapDomToView( root );
+
+			if ( isEnabled ) {
+				viewWriter.removeClass( 'ck-resizers-hidden', viewRoot );
+			} else {
+				viewWriter.addClass( 'ck-resizers-hidden', viewRoot );
+			}
```

Comment:
Also the class name should be more precise to better reflect the scope it affects - right now it may suggest that e.g. image resizers will be hidden too.